src/main/java/com/sk89q/craftbook/util/EntityType.java

```diff
@@ -26,7 +26,7 @@ public boolean is(Entity entity) {
             case MOB_PEACEFUL:
                 return entity instanceof Animals && !(entity instanceof HumanEntity);
             case MOB_ANY:
-                return entity instanceof Creature && !(entity instanceof HumanEntity);
+                return entity instanceof Mob && !(entity instanceof HumanEntity);
```

Comment:
You're going to need to explain *why* this change is being made. This is a pretty large change, and as far as I can tell incorrect?

Reply:
If you use the Creature class, it doesn't include Slimes because they directly extend from Mob instead of Creature. This still works with all friendly and hostile living entities that extend Creature though because Creature extends Mob.

So if you set a IC trap and use mob_any, it won't kill slimes with how it is currently